src/passkey_child/passkey_child_credentials.c

```diff
@@ -80,7 +80,13 @@ prepare_credentials(struct passkey_data *data, fido_dev_t *dev,
         goto done;
     }
 
-    ret = sss_generate_csprng_buffer(userid, sizeof(userid));
+    if (data->user_id == NULL) {
+        DEBUG(SSSDBG_OP_FAILURE, "user_id must be allocated using it.\n");
```

Comment:
Hi,

do you mean `... must be allocated before using it` ?

bye,
Sumit

Reply:
yes. fixed!